fenix/app/src/main/java/org/mozilla/fenix/debugsettings/navigation/DebugDrawerRoute.kt

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.debugsettings.navigation
+
+import androidx.annotation.StringRes
+import org.mozilla.fenix.R
+import org.mozilla.fenix.debugsettings.tabs.TabTools
+
+/**
+ * The navigation routes for screens within the Debug Drawer.
+ *
+ * @property route The navigation route of the destination.
+ * @property title The string ID of the destination's title.
+ */
+enum class DebugDrawerRoute(val route: String, @StringRes val title: Int) {
+    /**
+     * The navigation route for [DebugDrawerHome].
```

Comment:
```suggestion
     * The navigation route for [Home].
```

Reply:
This is intentionally `DebugDrawerHome`, so we can CMD+click into the source code (I think I just need to add the import for it).

Reply:
We could always typealias it, if we wanted

Reply:
Gotcha, I think the import is fine 🙂